pkg/process/metadata/parser/service.go

```diff
@@ -41,21 +42,33 @@ var _ metadata.Extractor = &ServiceExtractor{}
 
 // ServiceExtractor infers a service tag by extracting it from a process
 type ServiceExtractor struct {
-	enabled      bool
-	serviceByPID map[int32]*serviceMetadata
+	enabled               bool
+	useWindowsServiceName bool
+	serviceByPID          map[int32]*serviceMetadata
+	scmReader             *scmReader
 }
 
 type serviceMetadata struct {
 	cmdline        []string
 	serviceContext string
 }
 
+type WindowsServiceInfo struct {
```

Comment:
Nit: may want to add a small comment here.

Reply:
Added!